sst/test/sstconfigs/fastL1WithParams_config.py

```diff
@@ -0,0 +1,75 @@
+import sst
+import sys
+import os
+
+DEBUG_L1 = 0
+DEBUG_MEM = 0
+DEBUG_LEVEL = 10
+
+heap = ""
+clw = "8"
+if len(sys.argv) > 3:
+    heap = sys.argv[3]
+if len(sys.argv) > 4:
+    clw = sys.argv[4]
+
+cpu = sst.Component("core", "sstsimeng.simengcore")
+cpu.addParams({
+    "simeng_config_path": os.getcwd() + "/../../../configs/sst-cores/a64fx-sst.yaml",
```

Comment:
We've spoken about this offline but thought I'd place this comment here so it's trackable. We should try and make this path more agnostic to the user to avoid having to set the path manually

Reply:
Addressed in the most recent change, please review.

Reply:
Shouldn't this be resolved?

Reply:
I'm running the testing framework on my local system to ensure this works. I will resolve this once I can confirm it does.